README.md

```diff
@@ -1 +1,36 @@
-# INF 6420 - Intro to Web Development
+# Emily Morris's Media Portfolio 
+
+A portfolio showcasing my writing work.
+
+## Wireframes 
+
+![wireframe of Landing Page](wireframes/IMG_1509.jpg)
+
+Header - navigation menu and my name 
+
+Navigation Menu - links to other pages: contact and writing by publication 
+
+Main Content - brief overview of my experience and goals
+
+Footer - Copyright information 
+
+![wireframeofContactPage]
```

Comment:
The image source is not pointing to the correct location. Instead of `wireframeofContactPage` it should be `wireframes/name-of-file.jpg`.